gtclang/test/atlas-integration-test/CMakeLists.txt

```diff
@@ -0,0 +1,78 @@
+##===------------------------------------------------------------------------------*- CMake -*-===##
+##                          _                      
+##                         | |                     
+##                       __| | __ ___      ___ ___  
+##                      / _` |/ _` \ \ /\ / / '_  | 
+##                     | (_| | (_| |\ V  V /| | | |
+##                      \__,_|\__,_| \_/\_/ |_| |_| - Compiler Toolchain
+##
+##
+##  This file is distributed under the MIT License (MIT). 
+##  See LICENSE.txt for details.
+##
+##===------------------------------------------------------------------------------------------===##
+
+if(eckit_DIR AND atlas_DIR)
+
+    find_package(eckit)
```

Comment:
why is this insde the `if(eckit_DIR`?

Reply:
Will change to checking if `ENABLE_ATLAS_INTEGRATION_TESTS` is set to on